docs/source/upgrade/index.rst

```diff
@@ -91,27 +91,28 @@ Upgrade the ScyllaDB Manager Server and Client
 
 Confirm that the settings for ``scylla-manager.repo`` are correct.
```

Comment:
What does it mean that the settings are correct? What if they are not?
Does it refer to the following step, i.e., "confirm the settings are correct"="check if the version is as expected"?

I'd like to clarify this section, but I'm not sure how.

Reply:
Looks that we are not using `scylla-manager.repo` name in other parts of instruction.
This file keeps the definition of the repository which is used to install scylla-manager packages.

By `confirm that the settings for scylla-manager.repo are correct` I understand that the user should make sure that this repo file keep the definition of expected scylla-manager version.

Take a look here https://www.scylladb.com/download/?platform=ubuntu-22.04&version=manager-3.1#manager
In **2. Procedure  / Manager 3.1** there is the instruction of how to download the expected file.